discovery_child_development/pipeline/labelling/detection_management/testing.ipynb

```diff
@@ -0,0 +1,493 @@
+{
```

Comment:
Github is now able to render jupyter notebooks in the review process if the outputs are cleared and the notebook isn't super long, if you wanted it to be reviewed? I had a quick look at the file and all seems good! So no need to.

Reply:
Thanks, no need to review the notebook, it was just used for development.